kv/main.go

```diff
@@ -420,11 +491,12 @@ func main() {
 	var lastOps uint64
 	writers := make([]*blocker, *concurrency)
 
+	seq := &sequence{val: *writeSeq, seed: *seqSeed}
```

Comment:
can you log the initial and last (in `cast <-done`) sequence, that would make it easier to resume.